gef.py

```diff
@@ -7463,7 +7372,7 @@ def do_invoke(self, argv: List, *args, **kwargs) -> None:
             # Special (e.g. segment) registers go on their own line
             if regname in gef.arch.special_registers:
                 special_line += f"{Color.colorify(regname, color)}: "
-                special_line += f"{get_register(regname):#04x} "
+                special_line += f"0x{gef.arch.register(regname):04x} "
```

Comment:
why did you use `0x` this time?

Reply:
I don't think I did. Been rebasing so much on this PR that I'm not sure where this is coming from.